### DISCUSSION TITLE
No authentication strategy available

### DISCUSSION THREAD

**Discussion by leofass** [requarks/wiki#3070] [Help / Questions]
**Describe the bug**
There are no authentication available under 'Authentication' menu.

**To Reproduce**
1. Go to 'Administration > Authentication' menu
2. Click on 'Active' toggle
3. Scroll down to 'This strategy is active'
4. See error: "This strategy has no configuration options you can modify."

**Expected behavior**
Is expected to be available Google authentication method on this page, but an error occurs.

**Screenshots**
![Screenshot_20210203_105400](https://user-images.githubusercontent.com/8985359/106757622-5619b500-660f-11eb-9420-19a82bf4465d.png)

**Host Info (please complete the following information):**
 - OS: Docker Linux
 - Wiki.js version: 2.5.188
 - Database engine: Postgres 11.10

**Comment by NGPixel** [comment]
Is this a brand new installation? or did you just perform an upgrade?

**Comment by leofass** [comment]
@NGPixel No this is a upgraded stack. We did this upgrade since version 2 of docker image...

**Comment by NGPixel** [comment]
The screenshot you posted is definitely not 2.5. Try clearing your browser cache?
